games/o.yaml

```diff
@@ -196,6 +196,33 @@
   video:
     youtube: vnUIWB7Mixw
 
+- name: Open Greedy
+  originals:
+  - Pac-Man
+  type: remake
+  repo: 'https://github.com/atroel/open-greedy/'
+  url: 'http://troel.net/opengreedy/'
+  development: halted
+  status: playable
+  content: free
+  lang:
+  - C
+  framework:
+  - SDL
```

Comment:
I think this project use SDL2: https://github.com/atroel/open-greedy/blob/master/Build-linux.mk#L20